components/match2/wikis/starcraft/get_match_group_copy_paste_wiki.lua

```diff
@@ -40,37 +42,50 @@ end
 
 --returns the Code for a Match, depending on the input
 function wikiCopyPaste.getMatchCode(bestof, mode, index, opponents, args)
-	local score = args.score == 'true' and '|score=' or ''
-	local out = '{{Match' .. (index == 1 and bestof ~= 0 and ('|bestof=' .. bestof) or '') ..
-		(bestof == 0 and '\n\t|winner=' or '') .. '\n\t|date=\n\t|twitch='
+	local indent = '    '
+
+	if bestof == 0 and args.score ~= 'false' then
+		args.score = 'true'
+	end
+
+	local score = args.score == 'true' and '|score=' or nil
+	local lines = Array.extend(
+		'{{Match',
+		index == 1 and ('|bestof=' .. (bestof ~= 0 and bestof or '')) or nil,
+		score and indent .. score,
+		args.needsWinner == 'true' and indent .. '|winner=' or nil,
+		args.hasDate == 'true' and {indent .. '|date=', indent .. '|twitch='} or {}
+	)
 
 	for i = 1, opponents do
-		out = out .. '\n\t|opponent' .. i .. '=' .. wikiCopyPaste._getOpponent(mode, score)
+		table.insert(lines, indent .. '|opponent' .. i .. '=' .. wikiCopyPaste._getOpponent(mode, score))
 	end
 
 	if bestof ~= 0 then
 		if mode == 'team' and tonumber(args.submatch or '') then
 			local submatchBo = tonumber(args.submatch)
 			for i = 1, bestof do
-				local submatchNumber = math.floor((i-1)/submatchBo) + 1
-				out = out .. '\n\t|map' .. i .. '={{Map|map=|winner=|t1p1=|t2p1=|subgroup=' .. submatchNumber .. '}}'
+				local submatchNumber = math.floor((i - 1) / submatchBo) + 1
+				table.insert(lines, indent .. '|map' .. i .. '={{Map|map=|winner=|t1p1=|t2p1=|subgroup=' .. submatchNumber .. '}}')
```

Comment:
This is not a tab/space conversion.

Reply:
hmm? what do you mean?

Reply:
the only thing that's not a tab -> space conversion is at the beginning of `wikiCopyPaste.getMatchCode` - I added some stuff that was present in the sc2 version that I assume was left out in sc1 by accident

Reply:
Could you add a mention of that to the description?

Reply:
ok, added a bit in the description